src/components/HomeCategory/Item.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+interface HomeCategoryItem {
+  readonly imagePath?: string
+  readonly alternate?: string
+  readonly caption?: string
+}
+
+const HomeCategoryItem = ({imagePath, alternate, caption}: HomeCategoryItem) => {
+  return (
+    <div className="home-category__item">
+      <figure className="home-category__image-container">
+        <img
+          src={imagePath}
+          alt={alternate}
+          className="home-category__image"
+        />
```

Comment:
I would consider using `Image` component from [next/image](https://nextjs.org/docs/api-reference/next/image) instead.
Read more [here](https://nextjs.org/docs/basic-features/image-optimization)

Reply:
According to this [issue](https://github.com/vercel/next.js/discussions/18739), there are still problems that Next `Image` can't fit its container. Thus, we can use normal `<img/>` for now until the issue is fixed.